server/sonar-db-dao/src/main/java/org/sonar/db/organization/OrganizationDao.java

```diff
@@ -96,6 +97,17 @@ public void setDefaultTemplates(DbSession dbSession, String uuid, DefaultTemplat
     getMapper(dbSession).updateDefaultTemplates(uuid, defaultTemplates, now);
   }
 
+  public Optional<Integer> getDefaultGroupId(DbSession dbSession, String organizationUuid) {
```

Comment:
what is the rationale to expose this methods? Why not create either:
* `Optional<GroupDto> groupDao.selectByOrganizationUuid(orgUuid)`
* `OrganizationDto selectByUuid(DbSession dbSession, String organizationUuid)` – then you have access to `OrganizationDto.getDefaultGroupId`